web/client/src/views/Scan.vue

```diff
@@ -97,15 +136,43 @@ export default Vue.extend({
 
       this.$store.commit("setNumber", formatNumber(this.inputNumber));
 
-      this.scanEvent.$emit("scan");
+      try {
+        const res = await axios.post(`${config.apiUrl}/v2/numbers`, {
+          number: this.$store.state.number,
+        });
+
+        this.localData = res.data;
 
-      this.scanEvent.$on("finished", () => {
-        this.loading = false;
-      });
+        if (this.localData.valid) {
+          this.getScanners();
+          this.isLookup = true;
+        }
+      } catch (error) {
+        this.$store.commit("pushError", { message: error });
+      }
+
+      this.loading = false;
     },
     onSubmit(evt: Event) {
       evt.preventDefault();
     },
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    updateInputNumber(val: any) {
```

Comment:
Instead of `any`, isn't there a type we can use from the `VuePhoneNumberInput` component ?

Reply:
Nope, but i know what is in the object anyway, I created an interface